syschdemd.nix

```diff
@@ -1,19 +1,13 @@
 { lib, pkgs, config, defaultUser, ... }:
 
-let
-  nixpkgs = import <nixpkgs> {};
-
-  inherit (nixpkgs) daemonize;
-in
 pkgs.substituteAll {
   name = "syschdemd";
   src = ./syschdemd.sh;
   dir = "bin";
   isExecutable = true;
 
-  buildInputs = [ daemonize ];
+  buildInputs = with pkgs; [ daemonize ];
 
-  inherit daemonize;
```

Comment:
This line is still required (respectively needs to be changed to `inherit (pkgs) daemonize;`), otherwise `@daemonize@` is not substituted in the result:

```
>  wsl -d NixOS
/nix/store/723ndl2p1p7rlm9dbiplglk04svbz82x-syschdemd/bin/syschdemd: line 19: @daemonize@/bin/daemonize: No such file or directory
```